test/models/fs/message_draft_test.rb

```diff
@@ -7,8 +7,17 @@ class Fs::MessageDraftTest < ActiveSupport::TestCase
   include ActionDispatch::TestProcess::FixtureFile
 
   test "create_and_validate_with_fs_form method schedules Fs::ValidateMessageDraftJob" do
+    fs_api = Minitest::Mock.new
+    fs_api.expect :parse_form, {
+      "subject" => "1122334455",
+      "form_identifier" => "3055_781"
+    },
+    [Base64.strict_encode64(file_fixture("fs/dic1122334455_fs3055_781__sprava_dani_2023.xml").read)]
+
+  FsEnvironment.fs_client.stub :api, fs_api do
```

Comment:
Rozmýšľam či toto nezačne tiež blikať keď medzičasom pobeží nejaký iný test vo vedľajšom vlakne a vytiahne to

Reply:
Vieme to nejako jednoducho poriesit?

Reply:
Počkajme kým začne blikať potom vymyslíme. Jednoduchý fix nemáme